video/getting-started/setup-local-media/setup-local-media.js

```diff
@@ -1,34 +1,47 @@
-// If you are requiring twilio-video from CommonJS,
-//
-//     const Video = require('twilio-video');
-//
-//  If you are including twilio-video.js from a <script> tag,
-//
-//     const Video = Twilio.Video;
-//
+const {
+  createLocalTracks,
+  createLocalAudioTrack,
+  createLocalVideoTrack
+} = require('twilio-video');
 
-const localMedia = new Video.LocalMedia();
+var localTracks;
 
-// You can call getUserMedia and add the resulting MediaStream to localMedia.
-Video.getUserMedia().then(mediaStream => {
-  localMedia.addStream(mediaStream);
+// Create default local audio and video tracks
+createLocalTracks().then(localTracks => {
+  console.log('Got default audio and video tracks:', localTracks);
 });
 
-// Or you can add the microphone and camera independently.
-localMedia.addMicrophone().then(() => {
-  return localMedia.addCamera();
+// Create local tracks using constraints
+createLocalTracks({
+  audio: true,
+  video: { width: 640, height: 360 }
+}).then(localTracks => {
+  console.log('Got audio and video tracks with constraints:', localTracks);
 });
 
-// Remove the microphone.
-localMedia.removeMicrophone();
-
-// Remove the camera.
-localMedia.removeCamera();
+// Local track events
```

Comment:
@ktoraskartwilio On second thought, I think we should limit the scope of this section to just setting up the local tracks. Can you remove the `Local track events` section?